src/app/(BottomTabNavigation)/AllCases/EligibilityForm/[caseUid].tsx

```diff
@@ -14,7 +14,15 @@ export default function EligibilityForm() {
   const updateEligibility = async (status: Eligibility) => {
     if (caseUid !== undefined) {
       await updateCaseStatus(caseUid, status);
-      router.back();
+      if (status === Eligibility.INELIGIBLE) {
+        router.push({
+          pathname: 'AllCases/EligibilityForm/ConfirmEligibility',
+          params: { caseUid },
+        });
+      } else {
+        await updateCaseStatus(caseUid, Eligibility.ELIGIBLE);
+        router.back();
+      }
     }
   };
```

Comment:
Can we rewrite this function so it merely routes the users to the `ConfirmEligibility` screen. There are several `updateCaseStatus` calls being made here that I think we only make once they actually confirm their ineligibility.

Reply:
Only eligible users will be able to view this screen anyways.